data/human/culture republic.txt

```diff
@@ -0,0 +1,125 @@
+# Copyright (c) 2015 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+# "Missions" which add culture events to the spaceport.
+
+#mission "ZZCU cultural holiday template"
+#	minor
+#	source
+#		government "placeholder"
+#		attributes "placeholder"
+#		not attributes "station"
+#	to offer
+#		and
+#			month == placeholder
+#			day >= placeholder
+#			day <= placeholder
+#	on offer
+#		conversation
+#			`placeholder`
+#				decline
+			
+#mission "ZZCU cultural random template"
+#	minor
+#	source
+#		government "placeholder"
+#		attributes "placeholder"
+#		not attributes "station"
+#	to offer
+#		random == 1
+#	on offer
+#		conversation
+#			`placeholder`
+#				decline
+
+
+#############################################
+#      Republic with Syndicate Pairing      #
+#############################################
+
+mission "ZZCU Sad Gamblers -R"
+	minor
+	source
+		government "Republic"
+		attributes "near earth"
+		not attributes "station"
+	to offer
+		random == 1
+	on offer
+		conversation
+			`Passing through the spaceport you see a sizable group of people waiting on a lottery draw. You hear a whoop of success and your attention is drawn as one of them wins a moderate sum in an 12th division win (a day or two worth of standard pay).`
```

Comment:
a* 12th division win
Replace (a day or two worth of standard pay) with "which corresponds to a day or two of standard pay".

Reply:
Quotations would imply that someone spoke them? I think it would be more fitting to say `a moderate sum in an 12th division win; a day or two worth of standard pay.`

Reply:
That's what I get for changing it from an 'eighth' division win without proofreading.
Thanks vkr, went with that.